controllers/flinkcluster_converter.go

```diff
@@ -406,6 +364,58 @@ func getDesiredTaskManagerDeployment(
 	return taskManagerDeployment
 }
 
+// Gets the desired TaskManager deployment spec from a cluster spec.
+func getDesiredConfigMap(
```

Comment:
Since JM/TM deployment depend on the configmap, do we need to wait configmap to be ready first, then create deployments?

Reply:
The configMap will be created almost at the same time as the request, but there will be a slight delay since the pod's attempt to mount the configMap requires some sort of pod scheduling first. So is it not sufficient to request the configMap creation before JM, TM rather than controlling the order in the operator? Even if the configMap creation is delayed, to my knowledge, if the pod attempts to mount the configMap and fails because the configMap does not exist, it will repeat the mount retry. As far as I tested, the pod did not fail or restart in this process and kept its initial state "ContainerCreating".